cmd/kusk/internal/utils/githubclient_test.go

```diff
@@ -0,0 +1,36 @@
+package utils
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestGetTags(t *testing.T) {
```

Comment:
Why not just use the `github.com/stretchr/testify/assert` package?

Reply:
It is important as long it isn't not failing